pytorch/Dockerfile

```diff
@@ -100,6 +106,42 @@ RUN wget -q  --no-check-certificate https://raw.githubusercontent.com/oneapi-src
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/docker/third-party-programs-pytorch.txt -O /licensing/third-party-programs-pytorch.txt && \
     wget -q  --no-check-certificate https://raw.githubusercontent.com/intel/neural-compressor/master/LICENSE -O /licensing/LICENSE
 
+# Enable passwordless ssh for mpirun
+RUN mkdir /var/run/sshd
+RUN sed -i'' -e's/^#PermitRootLogin prohibit-password$/PermitRootLogin yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^#PasswordAuthentication yes$/PasswordAuthentication yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^#PermitEmptyPasswords no$/PermitEmptyPasswords yes/' /etc/ssh/sshd_config \
+        && sed -i'' -e's/^UsePAM yes/UsePAM no/' /etc/ssh/sshd_config \
+        && echo "Port "$SSHD_PORT"" >> /etc/ssh/sshd_config \
+        && echo "Host *" >> /etc/ssh/ssh_config \
+        && echo "  Port "$SSHD_PORT"" >> /etc/ssh/ssh_config \
+        && echo "  StrictHostKeyChecking no" >> /etc/ssh/ssh_config
+
+EXPOSE ${SSHD_PORT}
+CMD ["/usr/sbin/sshd", "-D"]
+
+RUN useradd -rm -d /home/intel -s /bin/bash intel && \
+    echo 'intel ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
+USER intel
+WORKDIR /home/intel
+RUN mkdir /home/intel/.ssh
+
+USER root
+RUN chown intel /etc/ssh/ssh_host_ecdsa_key && \
+   chown intel /etc/ssh/ssh_host_rsa_key && \
+   chown intel /etc/ssh/ssh_host_ed25519_key
+
+USER intel
+WORKDIR /home/intel
+RUN echo "cp "$SSH_KEY_PATH"/id_rsa /home/intel/.ssh/" >> ~/.bashrc \
+    && echo "cp "$SSH_KEY_PATH"/id_rsa.pub /home/intel/.ssh/" >> ~/.bashrc \
+    && echo "chmod 600 /home/intel/.ssh/id_rsa" >> ~/.bashrc \
+    && echo "chown intel /home/intel/.ssh/id_rsa" >> ~/.bashrc \
+    && echo "chown intel /home/intel/.ssh/id_rsa.pub" >> ~/.bashrc \
+    && echo "if [ ! -f /home/intel/.ssh/authorized_keys ]; then" >> ~/.bashrc \
+    && echo "  cat /home/intel/.ssh/id_rsa.pub > /home/intel/.ssh/authorized_keys" >> ~/.bashrc \
+    && echo "fi" >> ~/.bashrc
```

Comment:
You could copy the existing config from root instead of re-writing this code

Reply:
could you explain more? thanks

Reply:
`cp ~/.bashrc /home/intel/.bashrc && chown ... && chmod ...`
Replace all instances of `/home/intel` with `~`.

Reply:
understand, but don't want to also bring some local changes on ~/.bashrc into docker instance. might just stay with appending new codes into bashrc file in the docker instance.